npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -2,31 +2,53 @@
   margin-bottom: 40px;
 }
 
+.drag-and-drop__button {
+  margin-bottom: 32px;
+  margin-top: 65px;
+  padding-left: 15px;
+  position: static;
+  width: auto;
+}
+
+.drag-and-drop__button::before {
+  position: static;
+}
+
+.drag-and-drop__button--highlight {
+  background-color: $govuk-focus-colour;
+  box-shadow: 0 -2px $govuk-focus-colour, 0 4px govuk-colour('black');
+  color: $govuk-text-colour;
+  outline: none;
+}
+
+.drag-and-drop--dragover {
+  background: #b1b4b6;
+  outline-color: #6f777b;
```

Comment:
I just tried this in Firefox, and the outline doesn't seem to change colour for me. It stays black rather than switching to grey.

Reply:
Ah, yes; good spot! I changed the order of the CSS so the black one was overwriting the grey. I've now fixed it and checked it in Firefox and Chrome and they look fine now. Thanks.